example/lib/main.dart

```diff
@@ -5,6 +5,7 @@ import 'package:flutter_isolate/flutter_isolate.dart';
 import 'package:path_provider/path_provider.dart';
 import 'package:flutter_downloader/flutter_downloader.dart';
 
+@pragma('vm:entry-point')
```

Comment:
@nmfisher also please add that this `@pragma('vm:entry-point')` is needed to the changelog

Reply:
Added!